source/dub/commandline.d

```diff
@@ -678,7 +678,7 @@ class CommandArgs {
 	{
 		foreach (ref arg; m_recognizedArgs)
 			if (names == arg.names) {
-				assert(help_text is null);
+				enforce(help_text is null, format!("duplicated argument '%s' must not change helptext, consider to remove the duplication")(names));
```

Comment:
I think this is a good addition now, but it should stay an assert (it's a programming bug, not a runtime exception that may occur)

Reply:
true ...

Reply:
```suggestion
				assert(help_text is null, format!("duplicated argument '%s' must not change helptext, consider to remove the duplication")(names));
```